contracts/zero-ex/contracts/src/bridges/mixins/MixinKyber.sol

```diff
@@ -0,0 +1,117 @@
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/v06/LibERC20TokenV06.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IERC20TokenV06.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IEtherTokenV06.sol";
+
+/*
+    Kyber
+*/
+interface IKyberNetworkProxy {
+
+    /// @dev Sells `sellTokenAddress` tokens for `buyTokenAddress` tokens.
+    /// @param sellTokenAddress Token to sell.
+    /// @param sellAmount Amount of tokens to sell.
+    /// @param buyTokenAddress Token to buy.
+    /// @param recipientAddress Address to send bought tokens to.
+    /// @param maxBuyTokenAmount A limit on the amount of tokens to buy.
+    /// @param minConversionRate The minimal conversion rate. If actual rate
+    ///        is lower, trade is canceled.
+    /// @param walletId The wallet ID to send part of the fees
+    /// @return boughtAmount Amount of tokens bought.
+    function trade(
+        address sellTokenAddress,
+        uint256 sellAmount,
+        address buyTokenAddress,
+        address payable recipientAddress,
+        uint256 maxBuyTokenAmount,
+        uint256 minConversionRate,
+        address walletId
+    )
+        external
+        payable
+        returns(uint256 boughtAmount);
+}
+
+contract MixinKyber {
+
+    using LibERC20TokenV06 for IERC20TokenV06;
+
+    /// @dev Mainnet address of the WETH contract.
+    IEtherTokenV06 constant private WETH = IEtherTokenV06(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
```

Comment:
If we aren't doing immutables, we should put these in `DeploymentConstants` type contracts so we can override and test them.

Reply:
is there any reason to not use immutables?

Reply:
we should have a ^0.6 version of DeploymentConstants which uses immutables and the `chainid` opcode